src/components/tabs/tab.scss

```diff
@@ -56,6 +56,8 @@ body {
   overflow: hidden;
   user-select: none;
   list-style-type: none;
+  min-height: inherit;
```

Comment:
put ir 40px so it will work without getting a aditional margin at the bottom

Reply:
You're right, i've tested with `min-height: 42px` for the `tab__wrapper` it also worked to remove this little bar at the active one.